### PR TITLE
new multi-zone deployment guide

### PR DESCRIPTION
**Pull request by Bradamant3**
### Summary

- Break existing guide into two:
  - Conceptual/explanatory page (About multi-zone deployments)
  - How-to page (Set up a multi-zone deployment)
- Rework how-to page for conciseness, clarity. questions outstanding.

PTAL @nickolaev 

WIP, please do not merge. Not marked draft so we can get netlify preview and start async discussion in reviews.

Questions (in how-to file) begin on a newline with REVIEWER

### Testing

Preview build is here: https://deploy-preview-402--kuma.netlify.app/docs/1.1.1/deployments/how-to-multi-zone/

### CODE REVIEW

**Review by nickolaev — COMMENTED** [review]
First part, adding this and will continue with the rest later

File: docs/docs/1.1.1/deployments/how-multi-zone-works.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Components of a multi-zone deployment
+
+A multi-zone deployment includes different roles for different components of your mesh environment: 
+
+* The **global control plane** accepts connections only from remote control planes -- not from any data plane proxies. It provides configurations for the global [policies](/policies) that are applied to the data plane proxies.
+* The **remote control planes** accept connections from data plane proxies that are started in the same underlying zone, and they connect to the `global` control plane to fetch service mesh policies for the data plane proxies that connect to them.
+* The **data plane proxies** connect to the closest remote control plane in the same zone. 
+* The **ingress** data plane proxy is set up on each zone to provide cross-zone communication between services in different zones.
+
```
[the remark this answers is not in the document]
Comment (Bradamant3) [reply docs/docs/1.1.1/deployments/how-multi-zone-works.md:13]: Yes please! Are you volunteering?

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
+# Set up a multi-zone deployment
+
+For a description of how multi-zone deployments work in Kuma, see [about multi-zone deployments](../how-multi-zone-works). This page explains how to configure and deploy Kuma in a multi-zone environment:
+
+- Set up the global control plane
+- Set up the remote control planes
+- Verify control plane connectivity
+- Set up cross-zone communication between data plane proxies
+
+Before you start you should determine the zone names to use. You must assign the same `zone` value to every remote control plane you want to run in the same zone. This value is an arbitrary string.
```
[the remark this answers is not in the document]
Comment (Bradamant3) [reply docs/docs/1.1.1/deployments/how-to-multi-zone.md:10]: > Cna you rephrase that for:
> 
> * we can add zones at an arbitrary time later, no need to know their names before deploying global for example
> * using the same name is relevant only in Universal mode, in k8s it is automatic.

The example command for Kubernetes is
```
$ kumactl install control-plane \
  --mode=remote \
  --zone=<zone name> \
  --ingress-enabled \
  --kds-global-address grpcs://`<global-kds-address>` | kubectl apply -f -
```
Isn't the `zone` the only thing that associates all the bits you want to include in the same zone? How is it "automatic" for K8s?

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+Set the `controlPlane.mode` value to `global` in the chart, then install. On the command line, run:
+
+```sh
+$ helm install kuma --namespace kuma-system --set controlPlane.mode=global kuma/kuma
+```
+
+Or you can edit the chart and pass the file to the `helm install kuma` command.
+
+REVIEWER: IS THIS CLEARER? OR HAVE I GOTTEN THINGS WRONG/MORE OBSCURE?
```
[the remark this answers is not in the document]
Comment (Bradamant3) [reply docs/docs/1.1.1/deployments/how-to-multi-zone.md:47]: Don't you also need to get the external IP for the global control plane for Universal? We don't say anything about that one either ...

File: docs/docs/1.1.1/deployments/how-multi-zone-works.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* **DNS Resolver**: Kuma provides an out of the box DNS server on every `remote` control plane that will be used to resolve service addresses when estabilishing any service-to-service communication. It scales horizontally as we scale the `remote` control plane.
+* **Ingress Data Plane**: Kuma provides an out of the box `ingress` data plane proxy mode that will be used to enable traffic to enter a zone from another zone. It can be scaled horizontally. Each zone must have an `ingress` data plane deployed. This happens automatically with Kubernetes, although if you choose a Helm installation you need to add the ingress separately. With a Universal deployment, adding ingress is a separate step.
+
+
+* **Service Connectivity**: Automatically resolved via the built-in DNS resolver that ships with Kuma. When a service wants to consume another service, it will resolve the DNS address of the desired service with Kuma, and Kuma will respond with a Virtual IP address, that corresponds to that service in the Kuma service domain.
+
+When running in multi-zone mode, we introduce the notion of a `global` and `remote` control planes for Kuma:
```
[the remark this answers is not in the document]
Comment (Bradamant3) [reply docs/docs/1.1.1/deployments/how-multi-zone-works.md:24]: We need a definition of control plane, absolutely agree. But it belongs at a higher level, in one of the overview/introduction topics. I'll ponder fixing those up next 👯‍♀️

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nickolaev) [reply docs/docs/1.1.1/deployments/how-to-multi-zone.md]: Yes, kind of. With the universal, there are many options the user can choose to deploy.
So yes, hinting them to gather that info, but we can't possibly have anus specific instructions what the exact commands look like.

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickolaev) [reply docs/docs/1.1.1/deployments/how-to-multi-zone.md]: On K8s we have "replica"s of the same Control Plane. So the name and everything is just replicated for us. It is a K8s trait to have this automated for each application that needs to scale up/down.

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+  inbound:
+  - port: 2010
+    servicePort: 1010
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+With a Universal control plane, the service tag has no such limitation.
+
+REVIEWER: DOES THIS ^ APPLY TO A HYBRID ENVIRONMENT WHERE THE CONTROL PLANE RUNS IN UNIVERSAL MODE? DOES THIS REFER TO THE GLOBAL CONTROL PLANE? CAN WE CLARIFY THE DISTINCTION BETWEEN THESE TWO CASES (THAT IS, THE HYBRID AND THIS SECOND CASE WHICH MIGHT OR MIGHT NOT BE RELEVANT TO THE FIRST ONE)?
```
Comment (nickolaev) [inline docs/docs/1.1.1/deployments/how-to-multi-zone.md:259]: If there is a chance that the service is deployed on K8s that limitation will always apply. So let's try to enumerate here, which use-cases will have the limitation:
* all zones are K8s 
* hybrid mode, at least one zone is K8s. The service runs on that K8s zone(s)

Use-cases where the limitation does not apply:
* all zones are Universal
* hybrid mode, where the service runs ONLY in a universal zone (or many universal zones)

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+
+To consume a distributed service in a Universal deployment, where the application address is `http://localhost:20012`:
```
Comment (nickolaev) [inline docs/docs/1.1.1/deployments/how-to-multi-zone.md:261]: This is the "old" still supported mode to consumer services on Universal. We should put forward the Transparent Proxy mode here. Please take the content from here:
https://kuma.io/docs/1.1.1/networking/transparent-proxying/

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+:::
+::::
+
+The Kuma DNS service format (e.g. `echo-server_kuma-test_svc_1010.mesh`) is a composition of Kubernetes Service Name (`echo-server`),
+Namespace (`kuma-test`), a fixed string (`svc`), the service port (`1010`). The service is resolvable in the DNS zone `.mesh` where
+the Kuma DNS service is hooked.
```
Comment (nickolaev) [inline docs/docs/1.1.1/deployments/how-to-multi-zone.md:285]: This whole paragraph looks a bit misplaced.

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+
+### Delete a zone
+
+REVIEWER: CANNOT TELL WHAT THE COMMANDS ARE THAT YOU RUN, AND IN WHAT ORDER. THIS SECTION NEEDS MUCH HELP, PLEASE!
+
```
Comment (nickolaev) [inline docs/docs/1.1.1/deployments/how-to-multi-zone.md:290]: If the Global is K8s - delete the zone resource: something like `kubectl delete zone...." 
If Global is Universal using `kumactl delete zone ...`

**Review by nickolaev — COMMENTED** [review]
I guess we'll need a couple more passes.
I feel we still miss a bit of cross-zone security, will double check that.
Would like to see @jakubdyszkiewicz at least.

File: docs/docs/1.1.1/deployments/how-multi-zone-works.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (manibh-kong) [reply docs/docs/1.1.1/deployments/how-multi-zone-works.md:16]: @nickolaev please also explicitly mention that Kuma ingress is to be used only for cross zone communication but can not be used as an ingress for general traffic ingress like Kong Ingress controller. for example we can say something like following:
The **ingress** data plane proxy is set up on each zone to provide cross-zone communication between services in different zones. Kuma Ingress can not be used as a Kubernetes ingress controller, it's only for internal cross zone communication between multizone clusters.

File: docs/docs/1.1.1/deployments/how-to-multi-zone.md [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+To delete a `Zone` we must first shut down the corresponding Kuma remote control plane instances. As long as the Remote CP is running this will not be possible, and Kuma returns a validation error like:
+
+```
+zone: unable to delete Zone, Remote CP is still connected, please shut it down first
+```
+
+When the Remote CP is fully disconnected and shut down, then the `Zone` can be deleted. All corresponding resources (like `Dataplane` and `DataplaneInsight`) will be deleted automatically as well.
+
+### Disable a zpme
```
Comment (mmorel-35) [inline docs/docs/1.1.1/deployments/how-to-multi-zone.md:299]:
```suggestion
### Disable a zone
```

File: docs/docs/1.1.1/deployments/how-multi-zone-works.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+* The **ingress data plane proxy** Provides ingress to services in different zones. If a zone consumes services only, and does not provide them, the zone does not require this component.
+  * Configured automatically on Kubernetes if you install directly with `kumactl`.
+  * Must be explicitly enabled if you install on Kubernetes with Helm.
+  * Must be installed separately in Universal mode.
+
+## How it works
+
+Kuma manages service connectivity -- establishing and maintaining connections across zones in the mesh -- with the ingress data plane proxy and with a DNS resolver.
+
+The DNS resolver is part of each remote control plane. It resolves each service address to a virtual IP address for all service-to-service communication, and scales horizontally withi the remote control plane.
```
Comment (mmorel-35) [inline docs/docs/1.1.1/deployments/how-multi-zone-works.md:21]:
```suggestion
The DNS resolver is part of each remote control plane. It resolves each service address to a virtual IP address for all service-to-service communication, and scales horizontally within the remote control plane.
```

**Comment by lahabana** [comment]
Superseeded by #647

**Pull request closed without merging**